src/controllers/userController.js

```diff
@@ -0,0 +1,90 @@
+import { save, getById, update, getAutoSuggestUsers, getAllUsers, remove } from '../services/UserService';
+import { v4 as uuidv4 } from 'uuid';
+import { bad } from '../util/constant';
+import logger from '../util/logger';
+
+export const saveUser = async (request, response) => {
+    const userId = uuidv4();
+    const userLogin = request.body.login;
```

Comment:
Maybe it would be better to make DTO. I can save body, but if I do it, request.body.id will be set with user.id. It won't be secure. I did a halfway of making DTO. I also have a middleware to log  invoked service method (request.method) and arguments passed to it (https://github.com/eds832/nodejs-taks3.2/blob/task5/src/index.js#L12)